CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to the Lua Users Foundation
+
+This document serves as contribution guidelines and the points listed should be
+followed for contributions to the foundation.
+
+Contributions are expected to follow the [code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
Please drop this line until there is a content in that file. No point in confusing the reader with illusory complexity.